docs/spec_draft.md

```diff
@@ -343,6 +344,102 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_inference
+
+### Semantics
+
+Normalizes the `operand` tensor across batch and spatial dimensions, using
+`mean` and `variance` provided in input, for each feature in the `feature_index`
+dimension and produces a `result` tensor. More formally, the semantics can be
+expressed using MLIR syntax as follows:
+
+```mlir
+func.func @upsize(%0: tensor<2xf32>) -> tensor<2x2x2xf32> {
+  %1 = mhlo.reshape %0 : (tensor<2xf32>) -> tensor<1x2x1xf32>
+  %2 = "mhlo.broadcast_in_dim"(%1){
+    broadcast_dimensions = dense<[0,1,2]> : tensor<3xi64>
+  } : (tensor<1x2x1xf32>) -> tensor<2x2x2xf32>
+  func.return %2 : tensor<2x2x2xf32>
+}
+
+func.func @batch_norm_inference(
+    %operand: tensor<2x2x2xf32>,
+    %scale: tensor<2xf32>,
+    %offset: tensor<2xf32>,
+    %mean: tensor<2xf32>,
+    %variance: tensor<2xf32>,
+    %epsilon: tensor<f32>,
+    %feature_index: tensor<i64>
+    ) -> tensor<2x2x2xf32> {
+  // Assuming %feature_index = 1
```

Comment:
`feature_index` isn't used anywhere with the value `1`. Can you also explain how this applies with non-1 `feature_index` value?